### PR TITLE
Upgrade amp-ad support for Index Exchange

### PR DESCRIPTION
**Pull request by indexexchange**
Updating amp-ad type "ix" to support timeout and handling network errors

### CODE REVIEW

**Comment by ixpaciga** [comment]
@lannka - can you give an ETA on when this will be reviewed? Thanks.

File: ads/ix.js [hunk 1]
```diff
@@ -14,19 +14,39 @@
...
     writeScript(global, 'https://js-sec.indexww.com/indexJTag.js');
   } else { //DFP ad request call
+
+    let calledDoubleclick = false;
+    data.ixTimeout = isNaN(data.ixTimeout) ? DEFAULT_TIMEOUT : data.ixTimeout;
+    const timer = setTimeout(() => {
+      callDoubleclick(global, data);
+    }, data.ixTimeout);
+
+    function callDoubleclick(global, data) {
```
Comment (lannka) [inline ads/ix.js]: can you make it a function variable? also, seems both params are not needed anymore

**Comment by ixpaciga** [comment]
@lannka - thanks for the feedback, we've made those changes.

**Comment by ixpaciga** [comment]
@lannka - Can we get an ETA on when this will be reviewed again?

**Comment by jridgewell** [comment]
There's a merge conflict that needs to be resolved before the next review.

**Comment by ixpaciga** [comment]
I believe that cla/google warning is about me - yes I consent to my commits being contributed to this project.

@lannka - can I get an ETA on this being reviewed?

**Pull request merged**